sdk/translation/azure-ai-translation-document/src/main/java/com/azure/ai/translation/document/models/ListDocumentStatusesOptions.java

```diff
@@ -0,0 +1,424 @@
+     * requested via top (or top is not specified and there are more items to be
+     * returned), &#064;nextLink will contain the link to the next page.
+     * 
+     * 
+     * Clients MAY request server-driven paging with a specific page size by
+     * specifying a maxpagesize preference. The server SHOULD honor this preference
+     * if the specified page size is smaller than the server's default page size.
+     * 
+     * @return the maxpagesize value.
+     */
+    public Integer getMaxpagesize() {
+        return this.maxpagesize;
+    }
+
+    /**
+     * Set the maxpagesize property: maxpagesize is the maximum items returned in a page. If more items are
+     * requested via top (or top is not specified and there are more items to be
+     * returned), &#064;nextLink will contain the link to the next page.
+     * 
+     * 
+     * Clients MAY request server-driven paging with a specific page size by
+     * specifying a maxpagesize preference. The server SHOULD honor this preference
+     * if the specified page size is smaller than the server's default page size.
+     * 
+     * @param maxpagesize the maxpagesize value to set.
+     * @return the ListDocumentStatusesOptions object itself.
+     */
+    public ListDocumentStatusesOptions setMaxpagesize(Integer maxpagesize) {
+        this.maxpagesize = maxpagesize;
+        return this;
+    }
+
+    /**
+     * Get the documentIds property: documentIds to use in filtering.
+     * 
+     * @return the documentIds value.
+     */
+    public List<String> getDocumentIds() {
+        return this.documentIds == null ? null : new ArrayList<>(this.documentIds);
+    }
+
+    /**
+     * Set the documentIds property: documentIds to use in filtering.
+     * 
+     * @param documentIds the documentIds value to set.
+     * @return the ListDocumentStatusesOptions object itself.
+     */
+    public ListDocumentStatusesOptions setDocumentIds(List<String> documentIds) {
+        this.documentIds = documentIds == null ? null : new ArrayList<>(documentIds);
+        return this;
+    }
+
+    /**
+     * Get the statuses property: Statuses to use in filtering.
+     * 
+     * @return the statuses value.
+     */
+    public List<String> getStatuses() {
+        return this.statuses == null ? null : new ArrayList<>(this.statuses);
+    }
+
+    /**
+     * Set the statuses property: Statuses to use in filtering.
+     * 
+     * @param statuses the statuses value to set.
+     * @return the ListDocumentStatusesOptions object itself.
+     */
+    public ListDocumentStatusesOptions setStatuses(List<String> statuses) {
+        this.statuses = statuses == null ? null : new ArrayList<>(statuses);
+        return this;
+    }
+
+    /**
+     * Get the createdDateTimeUtcStart property: the start datetime to get items after.
+     * 
+     * @return the createdDateTimeUtcStart value.
+     */
+    public OffsetDateTime getCreatedDateTimeUtcStart() {
```

Comment:
Use `getCreatedAfter()` and `getCreatedBefore()` names to indicate the list fetches items that are created after/before the given datetime.

Reply:
Updated